doc/release-notes.md

```diff
@@ -1,10 +1,10 @@
-Dogecoin Core version 1.14.6 is now available from:
+Dogecoin Core version 1.14.7 is now available from:
 
-  <https://github.com/dogecoin/dogecoin/releases/tag/v1.14.6/>
+  <https://github.com/dogecoin/dogecoin/releases/tag/v1.14.7/>
 
-This is a new minor version release, including important security updates and
-changes to network efficiency. All Dogecoin Core users - miners, services,
-relay operators and wallet users - are strongly recommended to upgrade.
+This is a new minor version release, including important security updates for
+Dogecoin-Qt. Dogecoin-Qt users on any platform are strongly recommended to
+upgrade.
 
```

Comment:
It's because there is no urgency for anyone to update, except Qt users, as the security updates exclusively have the scope Dogecoin Qt, and nothing else. For everyone else, there is no urgent need to update unless they are urgently waiting for a feature, but then we don't recommend anything.

Recommending people to update should be done exclusively for security reasons. Any other reason to recommend someone to update is dangerous, because consensus gets formed by the binaries.

Reply:
I added a mention of RPC enhancements in the intro to make clear that it's not just Dogecoin Qt security updates.